dvc/scm/__init__.py

```diff
@@ -10,7 +10,8 @@ class NoSCM(Base):
     pass
 
 
-def SCM(root_dir):  # pylint: disable=invalid-name
+def SCM(root_dir, search_parent_directories=False):  # pylint:
+    # disable=invalid-name
```

Comment:
Looks like `black` decided to split it or was it you? :slightly_smiling_face: Not sure if this works after it is split.

Reply:
it was black, sorry